float64 x
float64 y
float64 sx
float64 sy
float64 svx
float64 svy
float64 sacc
